eamena/eamena/media/js/views/forms/location.js

```diff
@@ -102,10 +102,16 @@ define([
                 }
             }));
             
+            if (resourcetypeid == 'HERITAGE_FEATURE.E24') {
+                var siteshape_node = 'SITE_OVERALL_SHAPE_TYPE.E55';
+            } else { 
+                var siteshape_node = 'SITE_OVERALL_SHAPE_TYPE_NEW.E55';
```

Comment:
@mradamcox  the _NEW tag will disappear at the end of the data migration process (using the rename_entity_type utility: https://github.com/azerbini/eamena_dev/blob/15561846fab51b0a8dc89aebd3240d8e41482c26/arches/management/commands/package_utils/migrate_resources.py#L385) so this bit of code will become redundant then.